vite.config.mjs

```diff
@@ -2,7 +2,7 @@ import os from 'os';
 import { dirname, resolve } from 'path';
 import { fileURLToPath } from 'url';
 
-// vite.config.js
+// vite.config.mjs
```

Comment:
Renamed to `.mjs` since we're using this file as a valid module anyway to get rid of the CJS warning from the console.